presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -46,6 +46,7 @@ statement
     | INSERT INTO qualifiedName columnAliases? query                   #insertInto
     | DELETE FROM qualifiedName (WHERE booleanExpression)?             #delete
     | ALTER TABLE from=qualifiedName RENAME TO to=qualifiedName        #renameTable
+    | COMMENT ON TABLE qualifiedName IS string                         #commentTable
```

Comment:
Unfortunately, comments aren't part of the SQL standard at all. We chose this (slightly weird) syntax because it is the same for [Oracle](https://docs.oracle.com/database/122/SQLRF/COMMENT.htm), [PostgreSQL](https://www.postgresql.org/docs/9.6/static/sql-comment.html), [DB2](https://www.ibm.com/support/knowledgecenter/en/SS6NHC/com.ibm.swg.im.dashdb.sql.ref.doc/doc/r0000901.html) and [H2](http://www.h2database.com/html/grammar.html#comment). If it is ever standardized, this will very likely be the syntax.

MySQL uses `ALTER TABLE` (which doesn't allow setting a column comment without redefining the entire column) and SQLite seems not to support comments at all.